examples/http_server/module/src/lib.rs

```diff
@@ -37,37 +35,26 @@ impl Node<Invocation> for StaticHttpServer {
     fn handle_command(&mut self, invocation: Invocation) -> Result<(), OakError> {
         let request = invocation.receive()?;
 
-        info!("Handling a request to {}.", request.uri);
-
-        // Workaround for https://rust-lang.github.io/rust-clippy/master/index.html#borrow_interior_mutable_const.
-        static CONTENT_TYPE: http::header::HeaderName = http::header::CONTENT_TYPE;
-
-        let response = match request.uri.parse::<http::Uri>() {
-            Ok(uri) => match uri.path().as_ref() {
-                "/" => HttpResponse {
-                    body: include_bytes!("../static/index.html").to_vec(),
-                    status: http::StatusCode::OK.as_u16() as i32,
-                    headers: hashmap! {
-                        CONTENT_TYPE.as_str().to_string() => HeaderValue { values: vec!["text/html; charset=UTF-8".to_string().into_bytes()] },
-                    },
-                },
-                _ => HttpResponse {
-                    body: "not found".to_string().into_bytes(),
-                    status: http::StatusCode::NOT_FOUND.as_u16() as i32,
-                    headers: hashmap! {},
-                },
-            },
-            Err(err) => {
-                warn!("could not parse URI: {}", err);
-                HttpResponse {
-                    body: "invalid URI".to_string().into_bytes(),
-                    status: http::StatusCode::BAD_REQUEST.as_u16() as i32,
-                    headers: hashmap! {},
-                }
-            }
+        info!("Handling a request to {}.", request.uri());
+
+        let response = match request.uri().path().as_ref() {
+            "/" => http::response::Builder::new()
+                .status(http::StatusCode::OK)
+                .header(
+                    http::header::CONTENT_TYPE,
+                    "text/html; charset=UTF-8".to_string().into_bytes(),
```

Comment:
Is this conversion still necessary? https://docs.rs/http/0.2.1/http/response/struct.Builder.html#examples-3 the examples seem to suggest it is not.